### PR TITLE
96 biodetail

### PR DESCRIPTION
**Pull request by naina66406**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://96-biodetail--world-bank--aemsites.hlx.live/en/drafts/naina/bio-detail

### CODE REVIEW

File: blocks/bio-detail/bio-detail.css [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
-  font-size: var(--body-medium);
+.bio-detail .media-inquiries p a{
+  padding: 0;
+  margin: 0;
+  border-bottom: 2px solid transparent;
+  color: #000D1A;
+}
+
+.bio-detail .media-inquiries p a:hover{
+  color: #000D1A;
```
Comment (rrrragas) [inline blocks/bio-detail/bio-detail.css:16]: both normal and hover colors are same ? if so we dont need this

File: blocks/bio-detail/bio-detail.js [hunk 2]
```diff
@@ -19,7 +19,11 @@ function createSocialMediaLink(linkName, className, iconPath) {
     const linkImage = img({ class: className });
     linkImage.src = `${window.hlx.codeBasePath}/icons/${iconPath}`;
     anchor.appendChild(linkImage);
-    return div({ class: 'social-media-link' }, anchor);
+    const socialMediaLink = div({ class: 'social-media-link' }, anchor);
+    socialMediaLink.addEventListener('click', () => {
+      window.location.href = linkName.textContent;
```
Comment (rrrragas) [inline blocks/bio-detail/bio-detail.js:24]: current link clicks are going to wrong location.. is this authoring issue ? laso do we need this line code again if a link already has correct link in it's href

File: blocks/bio-detail/bio-detail.js [hunk 3]
```diff
@@ -72,18 +76,58 @@ function createPersonBio(
...
+    { name: 'surname', value: surName },
+    { name: 'upi-id', value: upiId },
+    { name: 'is-leader', value: isLeader },
+    { name: 'is-expert', value: isExpert },
+    { name: 'is-sr-manager', value: isSrManager },
+    { name: 'expertise-topics', value: expertiseTopics }];
+  listOfMetaTags.forEach((metaData) => {
+    const meta = document.createElement('meta');
+    meta.name = metaData.name;
+    meta.content = metaData.value.textContent;
```
Comment (rrrragas) [inline blocks/bio-detail/bio-detail.js:108]: plz have null checks

File: blocks/bio-detail/bio-detail.css [hunk 4]
```diff
@@ -1,12 +1,24 @@
...
+  border-bottom: 2px solid transparent;
+  padding: 4px 0;
+  font-weight: 400;
 }
 
-.bio-detail .resources ul li a {
-  color: var(--primary-blue-70);
-  text-decoration: none;
-  font-size: var(--body-medium);
+.bio-detail .media-inquiries p a{
```
Comment (thomsebastin) [inline blocks/bio-detail/bio-detail.css:8]: space after selector pls. `.bio-detail .media-inquiries p a{` to be `.bio-detail .media-inquiries p a {`

File: component-definition.json [hunk 5]
```diff
@@ -606,13 +606,17 @@
...
                   "job-title": "Senior Managing Director World Bank Group",
                   "x-link": "www.xlink.com",
                   "linkedin-link": "www.linkedin.com",
                   "instagram-link": "www.instagram.com",
                   "media-inquiries": "<p>&nbsp;<strong>Kimberly Versak</strong><br />Senior External Affairs Officer <br /> Tel: 202-458-8881 <br/ > kversak@worldbank.org",
-                  "resources": "<ul><li><a href='www.fullbio.com'>Full Bio</a></li><li><a href='www.highres.com'>High-Res Photo</a></li><li><a href='www.otherdownload.com'>Other Download</a></li></ul>"
+                  "resources": "<ul><li><a href='www.fullbio.com'>Full Bio</a></li><li><a href='www.highres.com'>High-Res Photo</a></li><li><a href='www.otherdownload.com'>Other Download</a></li></ul>",
+                  "is-leader": false,
+                  "is-expert": false,
+                  "is-sr-manager": false
```
Comment (thomsebastin) [inline component-definition.json:618]: `is-senior-manager` is more readable.

File: blocks/bio-detail/bio-detail.js (hunk 2 above)
Comment (naina66406) [reply blocks/bio-detail/bio-detail.js:24]: links were random. changed the links now. Pls check

File: blocks/bio-detail/bio-detail.css (hunk 1 above)
Comment (naina66406) [reply blocks/bio-detail/bio-detail.css:16]: on hover the color was is getting overridden from the styles file. so adding this @rrrragas

File: component-definition.json (hunk 5 above)
Comment (naina66406) [reply component-definition.json]: changed that @thomsebastin

**Comment by thomsebastin** [comment]
@naina66406 Lots of whitespace is getting added in meta tags. Could you please try to trim the text for any spaces before sending the text to append?

Screenshot 2024-09-12 at 3 47 07 PM

**Comment by naina66406** [comment]
trimmed the string @thomsebastin

**Comment by mkbansal1** [comment]
@naina66406 as discussed, please remove the meta properties as we have moved those properties in page properties.

**Comment by naina66406** [comment]
removed meta properties @mkbansal1

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**